src/components/Footer.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+
+function Footer() {
+    return (
+        <React.Fragment>
+            <div className="footer">
+                <div className style={{ textAlign: 'center' }}>
+                    <div
+                        style={{
+                        height: '40px',
+                        width: '40px',
+                        display: 'inline-block',
+                        margin: '24px',
+                        }}
+                    >
+                        <a href="https://www.facebook.com/groups/codeclannigeria/" target="_blank">
+                        <svg
+                            version="1.1"
+                            id="Layer_1"
+                            xmlns="http://www.w3.org/2000/svg"
+                            xmlnsXlink="http://www.w3.org/1999/xlink"
+                            x="0px"
+                            y="0px"
+                            viewBox="0 0 64 64"
+                            style={{ enableBackground: 'new 0 0 64 64' }}
+                            xmlSpace="preserve"
+                        >
+                            <g>
+                            <path
+                                style={{ fill: '#fff' }}
+                                d="M37.2,64H28c-2.5,0-4.5-2-4.5-4.5V37H17c-2.5,0-4.5-2-4.5-4.5v-6.9c0-2.5,2-4.6,4.5-4.6h6.5v-5.5c0-9.1,6.2-15.5,15-15.5
+                                H47c1.4,0,2.5,1.1,2.5,2.5v13c0,1.4-1.1,2.5-2.5,2.5h-5.5v3h5.4c0,0,0.1,0,0.1,0c1.4,0.1,2.7,0.7,3.5,1.8c0.8,1.1,1.2,2.4,0.9,3.8
+                                    l-1.3,6.7C49.8,35.4,48,37,45.7,37h-4.3v22.5C41.5,62,39.5,64,37.2,64z M28.5,59h8V34.5c0-1.4,1.1-2.5,2.5-2.5h6.3l1.2-6H39
+                                    c-1.4,0-2.5-1.1-2.5-2.5v-8c0-1.4,1.1-2.5,2.5-2.5h5.5V5h-6c-6.1,0-10,4.1-10,10.5v8c0,1.4-1.1,2.5-2.5,2.5h-8.5v6H26
+                                    c1.4,0,2.5,1.1,2.5,2.5V59z"
+                            />
+                        </g>
+                        </svg>
+                        </a>
+                    </div>
+ 
+                    <div
+                        style={{
+                        height: '40px',
+                        width: '40px',
+                        display: 'inline-block',
+                        margin: '24px',
+                        }}
+                    >
+                        <a href="https://github.com/codeclannigeria" target="_blank">
```

Comment:
Codacy found an issue: [Using target="_blank" without rel="noreferrer" is a security risk: see https://html.spec.whatwg.org/multipage/links.html#link-type-noopener](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=6058781)